File: official/docs/java/current/webhooks/create.java
package webhooks;

import com.easypost.exception.EasyPostException;
import com.easypost.model.Webhook;
import com.easypost.service.EasyPostClient;

import java.util.HashMap;

public class Create {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient(System.getenv("EASYPOST_API_KEY"));

        HashMap<String, Object> params = new HashMap<String, Object>();
        params.put("url", "example.com");

        Webhook webhook = client.webhook.create(params);

        System.out.println(webhook);
    }
}

File: official/docs/php/current/parcels/create.php
<?php

$client = new \EasyPost\EasyPostClient(getenv('EASYPOST_API_KEY'));

$parcel = $client->parcel->create([
    'length' => 20.2,
    'width' => 10.9,
    'height' => 5,
    'weight' => 65.9
]);

echo $parcel;
